frontend/src/components/regions/CardDetails/utils.js

```diff
@@ -0,0 +1,16 @@
+export const trimLongReview = (review) => {
+  return review.replace("\n", " ").split("\n").join(" ").replace("  ", " ");

```

Comment:
What about long reviews that dont have new line characters?

Reply:
Also this code is sketchy. First you replace the newlines with blank spaces, then try to split by new lines (there are no new lines. You just removed them) so you end up with an array of one element, which you then join up again (which is the same as just saying `array[0]`).

This also doesn't seem to trim anything. If the review has some well placed newline characters seperating some nice paragraphs or code examples, then this function will not trim things. It will just make things hard to read.

Please make sure that you look at this stuff in storybook. Take some long reviews with and without newline characters and actually look at what gets rendered.

Reply:
I shouldn't have pushed code with this function as I no longer used it